docker-compose/cassandra-3.11/README.md

```diff
@@ -0,0 +1,26 @@
+This directory provides two ways to start Stargate with Cassandra 3.11 using docker-compose.
+
+# Stargate with 3-node Cassandra 3.11 cluster
+
+Use the shell script `start_cass_311.sh`, which performs an orderly startup of containers 
+specified in `docker-compose.yml`:
+
+`./start_cass_311.sh`
+
+# Stargate with embedded Cassandra 3.11 (developer mode)
+
+This alternate configuration that runs the Stargate coordinator node in developer mode, so that no
+separate Cassandra cluster is required. This can be run directly with the command:
+
+`docker-compose -f docker-compose-dev-mode.yaml up`
```

Comment:
all `docker-compose` files have a `.yml` extension and not `.yaml`.. Either docs or files should be adapted..

Reply:
updated readme files.